setup.cfg

```diff
@@ -37,6 +37,10 @@ python_version = 3.7
 check_untyped_defs = True
 [mypy-tasks]
 ignore_errors = True
+[mypy-pysoa.common.serializer.base]
```

Comment:
Does this mean we're ignoring errors raised by mypy?

Reply:
Yes, there are a set of errors that are hard to resolve because the base class sets/user an attribute that is defined in the sub class. 

pysoa/common/serializer/base.py:36: error: "_SerializerMeta" has no attribute "mime_type"
pysoa/common/serializer/base.py:39: error: "_SerializerMeta" has no attribute "mime_type"
pysoa/common/serializer/base.py:41: error: "_SerializerMeta" has no attribute "mime_type"
pysoa/common/serializer/base.py:42: error: "_SerializerMeta" has no attribute "mime_type"
pysoa/common/serializer/base.py:45: error: "_SerializerMeta" has no attribute "mime_type"
pysoa/common/serializer/base.py:45: error: Incompatible types in assignment (expression has type "_SerializerMeta", target has type "Type[Serializer]")
pysoa/server/action/switched.py:65: error: "_SwitchedActionMetaClass" has no attribute "DEFAULT_ACTION"
pysoa/server/action/switched.py:69: error: "_SwitchedActionMetaClass" has no attribute "switch_to_action_map"
pysoa/server/action/switched.py:70: error: "_SwitchedActionMetaClass" has no attribute "switch_to_action_map"
pysoa/server/action/switched.py:71: error: "_SwitchedActionMetaClass" has no attribute "switch_to_action_map"
pysoa/server/action/switched.py:73: error: "_SwitchedActionMetaClass" has no attribute "switch_to_action_map"
Found 11 errors in 2 files (checked 167 source files)

Reply:
I see. I'm not an expert at all on mypy, but from what I understand PySOA has these type checks by design and it would be odd to me that we skip the checks we also write.

Also they look like fair errors, an expected property is not present in the class: https://github.com/eventbrite/pysoa/blob/f921735e54637c8692f9a338821add23c4c5bddb/pysoa/common/serializer/base.py#L36

Does this make sense for you?

Reply:
Yes, they are fair errors. Unfortunately to fix them we need to make code changes to PySOA.
At this point I don't want to re-design PySOA. I just need the build to pass. 
I am not confident on the current test suite to know that side effect to changes will be found.

Reply:
There doesn't seem to be a way to fix this without making code changes. 
At this point, suppressing the error seems a safer path forward.

Reply:
Ok, I see the point.

To confirm and out of curiosity this broke down because of a mypy breaking change, since we haven't change the productive code?

Reply:
Can we add a #TODO or ticket to review this later on?

Reply:
Added the TODO.